flink-shaded-guava-30/pom.xml

```diff
@@ -35,7 +35,7 @@ under the License.
     <packaging>jar</packaging>
 
     <properties>
-        <guava.version>30.1.1-jre</guava.version>
+        <guava.version>31.1-jre</guava.version>
```

Comment:
Since we update from 30 to 31 i guess we should also update name of shaded package from `${shading.prefix}.guava30.com.google` to `${shading.prefix}.guava31.com.google`
WDYT?

Reply:
> we should also update name of shaded package 

yes